.gitlab-ci.yml

```diff
@@ -49,6 +49,8 @@ workflow:
   image:
     name:                          "${CI_IMAGE}"
   before_script:
+    - rustup default stable
+    - rustup component add rustfmt clippy rust-src rustc-dev llvm-tools-preview
```

Comment:
I would add these `rust-src rustc-dev llvm-tools-preview` as prerequisites in the `README`. Previously the `toolchain` file installed them automatically.

Reply:
Although ideally once we remove the `dylint` driver build as part of the build here they will no longer be strict prerequisites, only required for the linting step

Reply:
Yes I will do this as a follow up. The branch is already prepared. No need to add stale README stuff.